proof/proof.go

```diff
@@ -63,6 +63,16 @@ type Proof struct {
 	// asset of the split.
 	SplitRootProof *TaprootProof
 
+	// MetaReveal is the set of bytes that were revealed to prove the
+	// derivation of the meta data hash contained in the genesis asset.
+	//
+	// TODO(roasbeef): use even/odd framing here?
```

Comment:
Yeah, basically if this should be even or odd or not. We'd need to go back and actually make this uniform if so. I think ultimately it makes sense to _force_ a reveal if the field is present, this way all the other merkle tree structures based on this would be canonical.